### ISSUE TITLE
OSVM 201804030952 crashes in Context(Object)>copy

### ISSUE THREAD

**Issue by fniephaus** [OpenSmalltalk/opensmalltalk-vm#265]
OSVM `201804030952` sometimes crashes in `Context(Object)>copy` when running Squeak SUnit tests.

Full stack trace at:
https://travis-ci.org/squeak-smalltalk/squeak-app/jobs/387262157#L2073

Another trace is at:
https://travis-ci.org/squeak-smalltalk/squeak-app/jobs/382950107#L1717

**Comment by smalltalking** [comment]
This one is probably related to a possible VM bug related to image segments. As a side effect, executing ImageSegmentTest will break the image.
It would make it easier to track down issues like this if the name of the executed test case appeared in the log.

**Comment by fniephaus** [comment]
Good point, I should turn that on, so we see what test cases are causing crashes...

**Comment by eliotmiranda** [comment]
Hi Fabio,

   I'd love to take a look at this.  But how do I go from
{
  "os": "osx",
  "script": "./prepare.sh",
  "language": "smalltalk",
  "osx_image": "xcode7.3",
  "smalltalk": "Squeak64-trunk"
}
to what's actually run on the command line?

**Comment by fniephaus** [comment]
@eliotmiranda have a look at [squeak-app](https://github.com/squeak-smalltalk/squeak-app). `prepare.sh` is the entry point and the builds fail [here](https://github.com/squeak-smalltalk/squeak-app/blob/d7cc75c4e7c9875c7eb5167e3ab46ae617eaed74/prepare_image.sh#L80). All files that are needed are in that repository. It's basically calling out to [smalltalkCI](https://github.com/hpi-swa/smalltalkCI) which then runs all SUnit tests with a [few exceptions](https://github.com/squeak-smalltalk/squeak-app/blob/d7cc75c4e7c9875c7eb5167e3ab46ae617eaed74/smalltalk-ci/Squeak-trunk.ston#L6). Otherwise, download the latest `Squeak-trunk` and run all tests a few times. Eventually, you'll get crashes like this.

**Comment by fniephaus** [comment]
CI builds still suffer from crashes caused by this, see https://github.com/hpi-swa/smalltalkCI/issues/440 for example. Anyone willing to look into this?
